advocacy_docs/partner_docs/ImpervaDataSecurityFabric/05-UsingImpervaDataSecurityFabric.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: 'Using Imperva Agent with EDB Postgres Advanced Server'
```

Comment:
Change to "Using Imperva Data Security Fabric with EDB Postgres Advanced Server"